### DISCUSSION TITLE
Saving and reusing netlist samples

### DISCUSSION THREAD

**Discussion by aovestdipaperino** [mamedev/discussions#101] [Ideas]
I was wondering if it's possible, for some drivers, to generate the samples ahead of time, especially those that require intensive calculations, to be reproduced when needed. Like an hybrid mode for less powerful HW (RPi). This could be an option that can be turned on in config or via driver switch.

**Comment by stonedDiscord** [comment]
I imagine that to be kinda difficult, especially with netlists that have a custom mixer and filtering after the mixer.

**Comment by aovestdipaperino** [comment]
I get the impression that the mixing part is there already. But all I see is loading samples from files. I would love to generate a prefilled memory array to do use.
